### PR TITLE
Change dev ports

### PR DESCRIPTION
**Pull request by soerenbf**
## Purpose

MacOS reserves port 5000 for OS. Also the docker compose workflow exposes postgres on 15432, so can we agree to assume this is the reasonable default?

Also updated readme's to match this reality.

### CODE REVIEW

**Review by schwartz-concordium — CHANGES_REQUESTED** [review]
Could you change to `repo1-path=/eks/testnet/timescaledb-alpha` in `pgbackrest.example.conf`

File: backend/README.md [hunk 1]
```diff
@@ -30,14 +33,16 @@ Schema is validated using snapshot testing in test `Tests.Api.GraphQL.GivenGraph
 The committed schema is saved as file `committed-schema.verified.graphql` and this file is used for frontend type generations.
 
 # Run the backend locally
-Once the prerequisites are met you can run the backend either from within the IDE or via a shell by issuing the "dotnet run" command in the Application directory.
+Once the prerequisites are met you can run the backend either: 
+- from within the IDE
+- via a shell by issuing the `dotnet run --project ./Application/Application.csproj` (from the `backend` folder)
```
Comment (schwartz-concordium) [inline backend/README.md:38]:
```suggestion
- via a shell by issuing `dotnet run --project ./Application/Application.csproj` (from the `backend` folder)
```

File: backend/README.md [hunk 2]
```diff
@@ -30,14 +33,16 @@ Schema is validated using snapshot testing in test `Tests.Api.GraphQL.GivenGraph
...
 
 ## Configuring the application for local execution
 The configuration file ``appsettings.Development.json`` contains the configuration values used when executing the application locally.
 
-* **PostgresDatabase -> ConnectionString**: The connection string to the main database for the backend
-* **PostgresDatabase -> ConnectionStringNodeCache**: The connection string to the Concordium Node cache
-* **ConcordiumNodeGrpc**: The configuration values that determine which Concordium Node is used when importing data
+* **PostgresDatabase -> ConnectionString**: The connection string to the main database for the backend. By default points to the postgres exposed by `../timescaledb-restore`
+* **PostgresDatabase -> ConnectionStringNodeCache**: The connection string to the Concordium Node cache. By default points to the postgres exposed by `../timescaledb-restore`
```
Comment (schwartz-concordium) [inline backend/README.md:44]:
```suggestion
* **PostgresDatabase -> ConnectionStringNodeCache**: The connection string to the Concordium Node cache. By default points to postgres exposed by `../timescaledb-restore`
```

File: backend/README.md [hunk 3]
```diff
@@ -30,14 +33,16 @@ Schema is validated using snapshot testing in test `Tests.Api.GraphQL.GivenGraph
...
+- from within the IDE
+- via a shell by issuing the `dotnet run --project ./Application/Application.csproj` (from the `backend` folder)
 
 ## Configuring the application for local execution
 The configuration file ``appsettings.Development.json`` contains the configuration values used when executing the application locally.
 
-* **PostgresDatabase -> ConnectionString**: The connection string to the main database for the backend
-* **PostgresDatabase -> ConnectionStringNodeCache**: The connection string to the Concordium Node cache
-* **ConcordiumNodeGrpc**: The configuration values that determine which Concordium Node is used when importing data
+* **PostgresDatabase -> ConnectionString**: The connection string to the main database for the backend. By default points to the postgres exposed by `../timescaledb-restore`
```
Comment (schwartz-concordium) [inline backend/README.md:43]:
```suggestion
* **PostgresDatabase -> ConnectionString**: The connection string to the main database for the backend. By default points to postgres exposed by `../timescaledb-restore`
```

File: backend/README.md [hunk 4]
```diff
@@ -5,13 +5,16 @@ This is the backend of Concordium Scan.
 ## Prerequisites
 The following prerequisites must be met in order to run the test suite and/or run the backend locally:
 
+* **Dotnet**
+    * .NET 6 SDK must be installed
```
Comment (schwartz-concordium) [inline backend/README.md:9]:
```suggestion
    * .NET 6 SDK must be installed. On macOS currently only dotnet x86 runtimes are supported due to its dependency to ConcordiumNetSdk.
```

**Review by schwartz-concordium — APPROVED** [review]

**Review by limemloh — APPROVED** [review]

**Pull request merged**